model/job.go

```diff
@@ -22,15 +23,15 @@ const (
 )
 
 type Job struct {
-	Id             string                 `json:"id"`
-	Type           string                 `json:"type"`
-	Priority       int64                  `json:"priority"`
-	CreateAt       int64                  `json:"create_at"`
-	StartAt        int64                  `json:"start_at"`
-	LastActivityAt int64                  `json:"last_activity_at"`
-	Status         string                 `json:"status"`
-	Progress       int64                  `json:"progress"`
-	Data           map[string]interface{} `json:"data"`
+	Id             string    `json:"id"`
+	Type           string    `json:"type"`
+	Priority       int64     `json:"priority"`
+	CreateAt       int64     `json:"create_at"`
+	StartAt        int64     `json:"start_at"`
+	LastActivityAt int64     `json:"last_activity_at"`
+	Status         string    `json:"status"`
+	Progress       int64     `json:"progress"`
+	Data           StringMap `json:"data"`
```

Comment:
This `StringMap` type is one of my pet peeves. Would you be able to use a `map[string]string` here? You'll need to add the following case the switch statement at line 788 of `store/sql_supplier.go` so that gorp can serialize it:
```go
	case map[string]string{}:
		return model.StringMapToJson(model.StringMap(t)), nil
```
And this case to the switch statement at line 835 of the same file to deserialize it:
```go
	case *map[string]string{}:
		binder := func(holder, target interface{}) error {
			s, ok := holder.(*string)
			if !ok {
				return errors.New(utils.T("store.sql.convert_string_map"))
			}
			b := []byte(*s)
			return json.Unmarshal(b, target)
		}
		return gorp.CustomScanner{Holder: new(string), Target: target, Binder: binder}, true
```